build.gradle

```diff
@@ -973,10 +973,28 @@ project(':iceberg-open-api') {
     testFixturesImplementation project(':iceberg-api')
     testFixturesImplementation project(':iceberg-core')
     testFixturesImplementation project(path: ':iceberg-core', configuration: 'testArtifacts')
-    testFixturesImplementation project(':iceberg-core').sourceSets.test.runtimeClasspath
```

Comment:
This brings too many things from test runtime class path and creates a problem for shadow jar. For example it brings some csv files, GPL license file etc.